examples/examples/revert.rs

```diff
@@ -29,7 +29,7 @@ async fn main() {
     let error = result_0.unwrap_err().inner;
     assert!(matches!(
         error,
-        ExecutionError::Revert(Some(reason)) if reason == "reason"
+        ExecutionError::Revert(Some(reason)) if reason == "revert: reason"
```

Comment:
Verified with a [test PR](https://github.com/cowprotocol/ethcontract-rs/pull/968) that this change is needed by now even without switching to `eth_chainId`.